src/Sylius/Bundle/ThemeBundle/Tests/Functional/Bundle/TestBundle/Resources/public/theme_asset.txt

```diff
@@ -1 +1 @@
-Vanilla asset
+Theme asset
```

Comment:
Or just tests were wrong and in fact only the `RELATIVE_SYMLINK` was used so they were passing because of that. If so, `AssetsInstaller` needs a fix.